README.md

```diff
@@ -40,6 +47,38 @@ Once your k8s cluster is started, you can execute the following script:
 KUBEVIRT_REPO=<kubevirt_repo_location> ./prepare_node.sh
 ```
 
+Once the cluster is deployed, and the macvtap CNI plugin is installed in the
+cluster nodes, the example scenario can be provisioned. That is done via
+kubectl:
+
+```bash
+kubectl create -f two_macvtap_fedora_vms.yaml
+```
+
+Afterwards, you'll have the virt-launcher pods and the vmi objects in the
+default namespace:
+
+```bash
+# list all pods in the default namespace
+kubectl get pods
+NAME                                  READY   STATUS    RESTARTS   AGE
+device-plugin-network-macvtap-44lbw   1/1     Running   0          3h8m
+device-plugin-network-macvtap-9pglq   1/1     Running   0          3h8m
+local-volume-provisioner-88sh9        1/1     Running   0          3h14m
+local-volume-provisioner-vkcr4        1/1     Running   0          3h14m
+virt-launcher-vm1-5jx2p               2/2     Running   0          14s
+virt-launcher-vm2-x2hm4               2/2     Running   0          14s
+
+# list all vmi objects in the default namespace
+kubectl get vmis
+NAME   AGE   PHASE     IP            NODENAME
+vm1    33s   Running   10.244.1.18   node02
+vm2    33s   Running   10.244.1.17   node02
+
+# connect to a vm via virtctl
+virtctl console --kubeconfig=<path to kubeconfig> vm1
```

Comment:
I am confused about the proper path to use for kubeconfig. Is this export done by _start_cluster.sh_ valid?

`export KUBECONFIG=$($KUBEVIRT_REPO_ROOT/cluster-up/kubeconfig.sh)`

I wasn't aware a script could be set for that. If so, probably we should advise here how to set it or just say that _start_cluster.sh_ does it.

Reply:
I think it's more explicit now. 

If not, please open an issue when you get back from vacations.